pkg/logger/logger.go

```diff
@@ -31,45 +26,3 @@ type Logger interface {
 	Warningf(format string, a ...interface{})
 	Failuref(format string, a ...interface{})
 }
```

Comment:
This seems like it could be moved to cli.go so that we dont have this file anymore? Having this separate feels like it would be getting used for both logger types.

Reply:
This is the interface definition, the other two files are the actual implementation. So yes, the point is having it referenced by the two logger types